libs/s25main/Debug.cpp

```diff
@@ -193,14 +184,12 @@ DebugInfo::~DebugInfo()
 
 std::vector<void*> DebugInfo::GetStackTrace(void* ctx) noexcept
 {
+#ifndef RTTR_CONTEXT_PTR_TYPE
+    using RTTR_CONTEXT_PTR_TYPE = void*;
+#endif
     std::vector<void*> stacktrace(256);
```

Comment:
hmm, what I recall is we probably should change this to a compile time storage (e.g std::array) since std::vector may not be able to allocate due to corruption/crash.

Reply:
True but it did work so far and we would likely run into the same issue in other places, e.g. with `std::stringstream`

I did convert this vector to a `boost::static_vector` but I haven't found a `static_allocator` or so in Boost so that change would be larger and not suitable for this PR.